Etg.Yams/Application/AppIdentity.cs

```diff
@@ -18,7 +18,13 @@ public AppIdentity(string id, Version version)
             _version = version;
         }
 
-        public override string ToString()
+		public AppIdentity(string id, string version)
+		{
+			_id = id;
+			_version = Version.Parse(version);
```

Comment:
What if the user doesn't use "n.n.n.n" version format?
Does `Version` supports Semantic Version?